cmd/clusterctl/cmd/config_cluster.go

```diff
@@ -65,50 +73,110 @@ var configClusterClusterCmd = &cobra.Command{
 
 		# Generates a yaml file for creating a Cluster API workload cluster with
 		# custom number of nodes (if supported by provider's templates)
-		clusterctl config cluster my-cluster --control-plane-machine-count=3 --worker-machine-count=10`),
+		clusterctl config cluster my-cluster --control-plane-machine-count=3 --worker-machine-count=10
+
+		# Generates a yaml file for creating a Cluster API workload cluster using a template hosted on a ConfigMap
+		# instead of using the cluster templates hosted in the provider's repository.
+		clusterctl config cluster my-cluster --from-config-map MyTemplates
+
+		# Generates a yaml file for creating a Cluster API workload cluster using a template hosted on specific URL
+		# instead of using the cluster templates hosted in the provider's repository.
+		clusterctl config cluster my-cluster --from https://github.com/foo-org/foo-repository/blob/master/cluster-template.yaml`),
```

Comment:
```suggestion
		clusterctl config cluster my-cluster --from ~/workspace/cluster-template.yaml,
```
Making the suggestion above because I think it provides a different example to the user and hints that the cluster-template can also be hosted locally. 
I made the mistake of considering the following when I read URL.
```
$ clusterctl config cluster my-cluster --from file://$HOME/workspace/cluster-template.yaml
Error: unable to read content from "file:///Users/wfernandes/workspace/cluster-template.yaml". Only reading from GitHub and local file system is supported
```

Reply:
@fabriziopandini I found the bug for the example above `file://...`. See https://github.com/kubernetes-sigs/cluster-api/pull/2265/files#r376556873 below. Lowercasing `File` gets it to work.

Reply:
ok I have added an additional example